Sources/NetworkProtection/NetworkProtectionDeviceManager.swift

```diff
@@ -301,14 +304,15 @@ public actor NetworkProtectionDeviceManager: NetworkProtectionDeviceManagement {
         return peerConfiguration
     }
 
-    func interfaceConfiguration(privateKey: PrivateKey, addressRange: IPAddressRange) -> InterfaceConfiguration {
-        var interface = InterfaceConfiguration(privateKey: privateKey)
-
-        interface.listenPort = 51821
-        interface.dns = [DNSServer(from: "10.11.12.1")!]
-        interface.addresses = [addressRange]
-
-        return interface
+    func interfaceConfiguration(privateKey: PrivateKey, addressRange: IPAddressRange, exclusions: [IPAddressRange], isKillSwitchEnabled: Bool) -> InterfaceConfiguration {
+        let dns = [
+            DNSServer(from: "10.11.12.1")!
```

Comment:
nit: This is fine for now. But at some point you should change this to use the value coming from the backend.
Just create a task for it and tackle it on a different PR.

Reply:
Yes, @diegoreymendez had a task for it

Reply:
That's correct on both comments.  But I'd say let's leave as is for the scope of this PR.